modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectViewLocalServiceImpl.java

```diff
@@ -238,6 +238,7 @@ public ObjectView updateObjectView(
 				newObjectViewColumn.setObjectFieldName(
 					objectViewColumn.getObjectFieldName());
 				newObjectViewColumn.setPriority(objectViewColumn.getPriority());
+				newObjectViewColumn.setLabelMap(objectViewColumn.getLabelMap());
```

Comment:
@p-albuquerque can you sort this too? Put right after `newObjectViewColumn.setObjectViewId`